BESMod/Systems/Demand/Building/BuildingsRoomCase600FF.mo

```diff
@@ -9,7 +9,7 @@ model BuildingsRoomCase600FF
     AZone={roo.AFlo},
     nZones=1);
   parameter Modelica.Units.SI.TemperatureDifference dTComfort=2
-    "Temperature difference to room set temperature at which the comfort is still acceptable. In DIN EN 15251, all temperatures below 22 °C - 2 K count as discomfort. Hence the default value. If your room set temperature is lower, consider using smaller values.";
+    "Temperature difference to room set temperature at which the comfort is still acceptable. In DIN EN 16798-1, all temperatures below 22 °C - 2 K count as discomfort. Hence the default value. If your room set temperature is lower, consider using smaller values.";
```

Comment:
Further down you use the international standard. Maybe you can refer to EN 16798-1:2019